Vagrantfile

```diff
@@ -160,6 +160,9 @@ end
 VAGRANTFILE_API_VERSION = "2"
 Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     config.vm.box_check_update = false
+    if Vagrant.has_plugin?("vagrant-vbguest")
+        config.vbguest.auto_update = false
```

Comment:
@jojimt: These three lines of code disable automatic update of the VirtualBox guest tools when this vbguest plugin is present. This plugin isn't needed for the netplugin repository, but some of our colleagues use this plugin.